src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7992,6 +8034,25 @@ internal static bool IsReparsePoint(FileSystemInfo fileInfo)
                 : Platform.NonWindowsIsSymLink(fileInfo);
         }
 
+        internal static bool IsNameSurrogateReparsePoint(string filePath)
+        {
+            if (Platform.IsWindows)
```

Comment:
Usually we use compile directives "#if UNIX".

Reply:
I realize that, but there's code above it that checks the platform.  I'm ok either way but was trying to keep it consistent with existing code in this file.